env.js

```diff
@@ -1,31 +1,41 @@
+/**
+ * JS file containing all env. and derived variables.
+ */
+
 const SERVICE_NAME = process.env.SERVICE_NAME || 'subsidy-application-management-service';
-const DATA_QUERY_CHUNK_SIZE = process.env.DATA_QUERY_CHUNK_SIZE || 50;
-const FORM_VERSION_DIRECTORY = process.env.FORM_DATA_DIR || '/share/versions/';
-const SEMANTIC_FORM_TYPE = process.env.FORM_DATA_DIR || 'http://lblod.data.gift/vocabularies/subsidie/ApplicationForm';
+const SERVICE_URI = `http://data.lblod.info/services/${SERVICE_NAME}`;
+const SEMANTIC_FORM_TYPE = process.env.SEMANTIC_FORM_TYPE ||
+    'http://lblod.data.gift/vocabularies/subsidie/ApplicationForm';
+const SEMANTIC_FORM_RESOURCE_BASE = process.env.SEMANTIC_FORM_RESOURCE_BASE ||
+    'http://data.lblod.info/application-forms/';
+
+const MUTATION_QUERY_CHUNK_SIZE = process.env.MUTATION_QUERY_CHUNK_SIZE || 50;
```

Comment:
Why `MUTATION` in the name ? Couldn't `QUERY_CHUNK_SIZE` be enough ?

Reply:
Because at this point is is only used by the n-triple-mutation service. But might be a good idea to generalize it for reuse by other future service!